src/org/folio/rest_v2/eureka/Eureka.groovy

```diff
@@ -212,4 +215,139 @@ class Eureka extends Base {
 
     return this
   }
+
+  /**
+   * Get Configured Tenants on Environment Namespace.
+   * @return Map of EurekaTenant objects.
+   */
+  Map<String, EurekaTenant> getExistedTenantsFlow() {
```

Comment:
Flow is something that doesn't return anything or just metadata
I think for this method will be good to leave simply getExistedTenants()

Reply:
Enhanced accordingly to your comment.